RageLib/Resources/ResourceHelpers.cs

```diff
@@ -322,6 +322,7 @@ public static void AssignPositions(IList<IResourceBlock> blocks, uint basePositi
                     baseSizeMaxTest *= 2;
                 }
 
+                flags = new ResourceChunkFlags(new uint[9], baseShift);

```

Comment:
What's this for?

Reply:
Ah I was trying to just make it a bit better readability wise by moving the flags down next to where they get used, I forgot to remove the above flags somehow...
Will fix